### PR TITLE
Go fish

### PR DESCRIPTION
**Pull request by kerenvr**
Go fish class determines to surrender card or to go fish.

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -156,3 +206,104 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
+
```
Comment (kerenvr) [inline source/CardGames.py:209]: my code

**Comment by Zwilson1301** [comment]
Looks good to me, great work!

File: source/CardGames.py [hunk 2]
```diff
@@ -20,9 +22,14 @@ def __init__(self, suit, value, image, cardBack):
     self.value = value
     self.image = image
     self.shortImage = []
-    if self.image:
```
Comment (Zwilson1301) [inline source/CardGames.py:23]: Did you mean to remove this line?

**Review by Zwilson1301 — APPROVED** [review]

**Review by psk44 — COMMENTED** [review]
The looping through the cards looks good. Displaying the player's hands and printing the statement looks good too! Can't wait to play!

**Pull request merged**